### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by jrperritt**

### CODE REVIEW

**Comment by jrperritt** [comment]
This could use some clean-up, but I wanted to get some code in here.

**Comment by jrperritt** [comment]
testing webhook

File: app.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+var express = require('express'),
+    exphbs  = require('express-handlebars');
+
+var app = express();
+app.engine('handlebars', exphbs());
+app.set('view engine', 'handlebars');
+
+app.get('/', function(req, presenterRes) {
+  // form route to query
+  mapping_route = normalize_url(mapping_service_url) + '/at/' + req.hostname + req.path;
```
Comment (kenperkins) [inline app.js]: I'd love to use a url module for joining paths here. Should `req.hostname` and `req.path` be using `encodeURIComponent`?

File: app.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  // query mapping service for content ID
+  http.get(mapping_route, function(mappingRes) {
```
Comment (kenperkins) [inline app.js]: Any motivation to not use a module like `request` for queries over the wire in lieu of the native `http` module?

Comment (jrperritt) [inline app.js]: Purely to minimize third-party dependencies. I'm indifferent.

File: app.js (hunk 1 above)
Comment (jrperritt) [inline app.js]: Yeah, those should probably be encoded.

File: app.js (hunk 2 above)
Comment (smashwilson) [inline app.js]: I'm :+1: on using a library. Node land is all about having many tiny modules that do one thing well, and shipping with Docker means we shouldn't have issues keeping development and production consistent even with many of them.

**Comment by smashwilson** [comment]
I'm going to merge this now so I have something to deploy and play around with. Thanks for kicking this off for us!

**Pull request merged**